spec/mailers/previews/user_mailer_preview.rb

```diff
@@ -0,0 +1,18 @@
+# Preview all emails at http://localhost:3000/rails/mailers/user_mailer
+class UserMailerPreview < ActionMailer::Preview
+
+  # Preview this email at http://localhost:3000/rails/mailers/user_mailer/account_activation
+  def account_activation
+    user = User.last
+    user.activation_token = User.new_token
+    UserMailer.account_activation(user)
+  end
+
+  # Preview this email at http://localhost:3000/rails/mailers/user_mailer/password_reset
```

Comment:
I suggest you to remove these comments. It will make the code more readable.

Reply:
I prefer to keep these ones for the quick access they provide when testing these views